cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -83,6 +83,7 @@ def self.config(env)
               /teacher-dashboard*
               /manage-professional-development-workshops*
               /professional-development-workshop-surveys*
+              /pd-program-registration*
```

Comment:
This is needed for `authentication_required!` to work (`index.haml, line 8 below)